Code/Editor/EditorPreferencesPageViewportCamera.cpp

```diff
@@ -77,6 +77,7 @@ void CEditorPreferencesPage_ViewportCamera::Reflect(AZ::SerializeContext& serial
         ->Field("PanInvertedX", &CameraMovementSettings::m_panInvertedX)
         ->Field("PanInvertedY", &CameraMovementSettings::m_panInvertedY)
         ->Field("DefaultPosition", &CameraMovementSettings::m_defaultPosition)
+        ->Field("DefaultOrientation", &CameraMovementSettings::m_defaultPitchYaw)
```

Comment:
Does it need a version bump up?

Reply:
Ah cool thanks! I'll leave as-is for now 👍